### DISCUSSION TITLE
What's the difference between Xpra, RDP and Moonlight/Sunshine?

### DISCUSSION THREAD

**Discussion by Mannshoch** [Xpra-org/xpra#4172] [Q&A]
What's the difference between Xpra, RDP and [Moonlight](https://github.com/moonlight-stream)/[Sunshine](https://github.com/LizardByte/Sunshine)?

**Comment by totaam** [comment]
* RDP is a protocol: https://en.wikipedia.org/wiki/Remote_Desktop_Protocol
* moonlight / sunshine are for streaming
* xpra does streaming, shadowing, seamless apps, multi-monitors, etc: https://github.com/Xpra-org/xpra/tree/master/docs/Features

**Reply by ehfd** [reply]
https://github.com/selkies-project/selkies-gstreamer is an HTML5 Linux counterpart to Moonlight/Sunshine adequate for streaming. We're associates.

**Comment by Mannshoch** [comment]
Obviously I do not ask if names are protocols.
Sorry for not being mor detailed. I ask because I'm looking for a good solution for a kind of houshold mainframe.

- Sunshine needs a "fake EDID" dongle ??? for using headless. Xpra also? (If yes, Why??) 
- how good does an application, may a game resize to the client Fullscreen?
- Could I create a Family server where three user could access apps on their profile the same time?

**Reply by ehfd** [reply]
You should see if you like https://github.com/selkies-project/docker-nvidia-glx-desktop and https://github.com/selkies-project/docker-nvidia-glx-desktop or not. We don't have simultaneous concurrent sessions, though.

**Comment by totaam** [comment]
No, xpra does not use or require dongles.

If you want to stream games, xpra is not your best choice.

Your last question seems to be about concurrent sessions? I see no problem with that.
